lib/ModelDataSourceAdapter.js

```diff
@@ -11,7 +11,8 @@ ModelDataSourceAdapter.prototype.set = function set(jsongResponse) {
 };
 
 ModelDataSourceAdapter.prototype.call = function call(path, args, suffixes, paths) {
-    var params = [path, args, suffixes].concat(paths);
```

Comment:
This creates two arrays - one for the literal, one from concat - when we only need one.